Packs/PrismaCloudCompute/Scripts/PrismaCloudLocalTrustedImagesListUpdate/PrismaCloudLocalTrustedImagesListUpdate.yml

```diff
@@ -0,0 +1,32 @@
+commonfields:
+  id: PrismaCloudLocalTrustedImagesListUpdate
+  version: -1
+name: PrismaCloudLocalTrustedImagesListUpdate
+script: ''
+type: python
+tags: []
+comment: 'Takes the results of "prisma-cloud-compute-images-scan-list" and "prisma-cloud-ci-scan-results-list" commands and creates or updates a list of trusted images which can be used for updating the "Trusted Images" list in Prisma Cloud, using the PrismaCloudRemoteTrustedImagesListUpdate automation.'
+enabled: true
+args:
+- name: list_name
+  required: true
+  description: The name of the XSOAR internal list we wish to update or create in case it doesn't exist.
+- name: deployed_images
+  description: The results of the "prisma-cloud-compute-images-scan-list" command.
+  isArray: true
+- name: passed_ci_scan_images
+  description: The "entityInfo" results of the "prisma-cloud-compute-ci-scan-results-list" command.
```

Comment:
Should this be: A comma-separated list of the .....

Reply:
The results of one command run might be a comma-separated list, the script works with the output of the command when it has any number of results (0, 1 and more).

Reply:
Leave it as is.